routes/crypto/products.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../../middlewares/authenticate')
+const productsController = require('../../controllers/crypto/productsController')
+
+/**
+ * @swagger
+ * /crypto/products:
+ *   post:
+ *     summary: Post product details to Database
+ *     tags:
+ *       - Crypto
+ *     parameters:
+ *       - in: body
+ *         data: productObject
+ *     responses:
+ *       201:
+ *         description: Product added successfully
+ *       409:
+ *         description: Product already exists
```

Comment:
Please add the schema for this error in `utils/swaggerDefinition.js` and write it the same way you wrote for error 401 below! 👍 
You can have a look at this - 
![image](https://user-images.githubusercontent.com/66248039/104125399-73df5b80-537c-11eb-9536-940fbb95f765.png)